circuit_knitting/cutting/cutting_experiments.py

```diff
@@ -165,25 +162,6 @@ def generate_cutting_experiments(
                 _append_measurement_circuit(new_qc, cog, inplace=True)
                 subexperiments_dict[label].append(new_qc)
 
-    # Remove initial and final resets from the subexperiments.  This will
```

Comment:
We may need to put this back in and replace the `do_while`. Leaving it removed for now to focus on other compatibility issues.